runelite-client/src/main/java/net/runelite/client/plugins/raids/RaidsPlugin.java

```diff
@@ -807,4 +907,50 @@ private void screenshotScoutOverlay()
 		imageCapture.takeScreenshot(overlayImage, "CoX_scout-", false, config.uploadScreenshot());
 		graphic.dispose();
 	}
+
+	private int findLobbyIndex(Point gridBase)
+	{
+		int x;
+		int y;
+		Tile[][] tiles = client.getScene().getTiles()[LOBBY_PLANE];
+
+		// if there is no room above the lobby then it is the north most room
+		// The north rooms will have a y value of 0 and the south will have a value of 1
+		if (tiles[gridBase.getX()][gridBase.getY() + RaidRoom.ROOM_MAX_SIZE] == null)
+		{
+			y = 0;
+		}
+		else
+		{
+			y = 1;
+		}
+
+		// if there is no room right of the lobby then it is the east most room
+		// The west most room will have a value of 0 and the east most will have a value of 3
+		if (tiles[gridBase.getX() + RaidRoom.ROOM_MAX_SIZE][gridBase.getY()] == null)
+		{
+			x = 3;
+		}
+		else
+		{
+			// determine x based on how many rooms are to the west of it.
+			for (x = 0; x < 3; x++)
+			{
+				int sceneX = gridBase.getX() - 1 - RaidRoom.ROOM_MAX_SIZE * x;
+				if (sceneX < 0 || tiles[sceneX][gridBase.getY()] == null)
+				{
+					break;
+				}
+			}
+		}
+
+		return x + (y * 4);
```

Comment:
Since this is a bitpacked int, we should use bitwise operations.
```suggestion
		return y << 2 | x;
```